prometheus/alertmanager/versions.json

```diff
@@ -4,43 +4,43 @@
+    "alertmanager-0.29.0.linux-armv6.tar.gz": "f5bdb05cb412682c174140f62a2677e21675f861e62420ca05cfec81cac97704",
+    "alertmanager-0.29.0.linux-armv7.tar.gz": "f5b7a2309763a0f8646b39af33c8778c1af989db939656ea1234142038d8998d",
+    "alertmanager-0.29.0.linux-mips.tar.gz": "fbe645d4d321de8e7eb35b6a986a0bcd302663ef14329672d944eb2ae275b9db",
+    "alertmanager-0.29.0.linux-mips64.tar.gz": "c835ddff6686f63812d59598efdf5de32590b1afd5c662942ea3b1159706c106",
+    "alertmanager-0.29.0.linux-mips64le.tar.gz": "ee26e1a7abafbaffc4aecefc5a7502bb5a8219b98742aa6cb66ac64cb099e594",
+    "alertmanager-0.29.0.linux-mipsle.tar.gz": "492893c21895fb7e2256b87ccdcd267a21d4a342e8202eabd3f9264eb552b48a",
+    "alertmanager-0.29.0.linux-ppc64.tar.gz": "44e47bbb5b59872e107168f5bfdb578874b257a3f5e4b2db58e522f28499f96b",
+    "alertmanager-0.29.0.linux-ppc64le.tar.gz": "6cacbadb7bb669df008f7a3a94c3eae99098d6af3667068bdac76a86263e5af8",
+    "alertmanager-0.29.0.linux-riscv64.tar.gz": "704ec75762609686bf345b0cd410ab99dcfb1b5508a4dc5778089eeb49b40535",
+    "alertmanager-0.29.0.linux-s390x.tar.gz": "f982300a1143903eaac226a8baddd225203364264f18e64642b273d4b436c325",
+    "alertmanager-0.29.0.netbsd-386.tar.gz": "326984e20320938dfc3c28c19f73a4978d7a77c0f3faca7352c1780dc80a52ee",
+    "alertmanager-0.29.0.netbsd-amd64.tar.gz": "0621a85ddcc6b66965abc28a31626feda158727f308253d4fdd96312bf44a5c6",
+    "alertmanager-0.29.0.netbsd-arm64.tar.gz": "c65a00bec957887bce47d2f44a0dc13fa91361300aaeacda9f9ae25442507ad8",
+    "alertmanager-0.29.0.netbsd-armv6.tar.gz": "00351564f1f793c7d28052d94a67a396da646dabd16883bf7f9384c5e1ad5327",
+    "alertmanager-0.29.0.netbsd-armv7.tar.gz": "56c6172390d9f6ce2fe5124f9ef0e968647499eaa635094a585f4d76afe9b76c",
+    "alertmanager-0.29.0.openbsd-386.tar.gz": "5a8d4521cb13a12c66a0bddd3faab0b7d6c918455bfabc3b9c530eaa851738d0",
+    "alertmanager-0.29.0.openbsd-amd64.tar.gz": "6555b8ef33b64f2baeb782065d0ed639987c42c159f5956011578f06c6c47ad2",
+    "alertmanager-0.29.0.openbsd-arm64.tar.gz": "08332642518e9751c48ef96233caad42ce79a132603df661058f70ebaeec07d6",
+    "alertmanager-0.29.0.openbsd-armv7.tar.gz": "d4a16471066bc6bc56ef9be02e72d129213f3d1f4aeca7251b3212a298a9bb34",
+    "alertmanager-0.29.0.windows-386.tar.gz": "5cade696deb2a56559de115b3b3fb47dba5afcd8ad0d9178455b2a46781097a3",
+    "alertmanager-0.29.0.windows-386.zip": "24d15ecc02c8a2cdc4c5fe9f6103a0ad5f75562b5af45c3c8f6bfb784c85ced9",
+    "alertmanager-0.29.0.windows-amd64.tar.gz": "d25b761de43741645dd094160ae2e10bea76f46c95ed3ce9b2af4aa9b3184883",
+    "alertmanager-0.29.0.windows-amd64.zip": "b08b0c741288131ab6fab0f98cc2637d8d1f315e224fb330cdbf02091db85604",
+    "alertmanager-0.29.0.windows-arm64.tar.gz": "f091dd7f299686e881e69d40276e99943602a7f0e7424edd9bba7f9896b6b5a1",
+    "alertmanager-0.29.0.windows-arm64.zip": "4adba683b38c81309827e55f0823ded0e966ab57990bd57785cd890e31d34a1b"
```

Comment:
heh, "oops" -- the reason for this change is documented in https://github.com/prometheus/alertmanager/issues/4680 :+1:
(they fixed/re-ran the release pipeline, and that republished all the binaries)